docs/sources/configure/_index.md

```diff
@@ -886,6 +886,28 @@ volume_results_cache:
   # CLI flag: -frontend.volume-results-cache.compression
   [compression: <string> | default = ""]
 
+# Cache instant metric query results.
+# CLI flag: -querier.cache-instant-metric-results
+[cache_instant_metric_results: <boolean> | default = false]
+
+# If a cache config is not specified and cache_instant_metric_results is true,
+# the config for the results cache is used.
+instant_metric_results_cache:
```

Comment:
```suggestion
instant_metric_query_results_cache:
```

to be consistent with `instant_metric_query_split_align`.

Reply:
same rationale like previously mentioned. Tried to be consistent with other results cache.

```
-bash-5.2$ rg "results_cache" | rg "yaml:\"" | awk '{print $4}'
`yaml:"results_cache,omitempty"`
`yaml:"index_stats_results_cache"
`yaml:"volume_results_cache"
`yaml:"instant_metric_results_cache"
`yaml:"series_results_cache"
`yaml:"label_results_cache"
`yaml:"results_cache"`
```